src/Options/index.js

```diff
@@ -194,6 +194,8 @@ export interface ParseServerOptions {
   /* Session duration, in seconds, defaults to 1 year
   :DEFAULT: 31536000 */
   sessionLength: ?number;
+  /* Default limit for the size of results set on queries, defaults to `100`. */
```

Comment:
```suggestion
  /* Default limit for the size of results set on queries.
  :DEFAULT: 100 */
```

Reply:
Why did you remove the info about the default value? How else is that visible in the docs?

Reply:
Amended